### ISSUE TITLE
Unable to create RequestInit for fetch

### ISSUE THREAD

**Issue by jbrownsw** [jodinathan/js_bindings#12]
```
import 'package:js_bindings/js_bindings.dart';

void main() {
  var init = RequestInit(
      method: 'GET',
      referrer: '',
      referrerPolicy: ReferrerPolicy.origin,
      mode: RequestMode.cors,
      credentials: RequestCredentials.omit,
      cache: RequestCache.noCache,
      redirect: RequestRedirect.follow,
      integrity: '',
      keepalive: true);
}
```

This results in:
`JSNoSuchMethodError (TypeError: Cannot read properties of undefined (reading '|staticInteropFactoryStub'))`

I'm using the latest package version 0.0.6.

**Comment by jodinathan** [comment]
it is a bug in the sdk, here is the [issue](https://github.com/dart-lang/sdk/issues/49243)

**Comment by jodinathan** [comment]
this is fixed with the SDK 2.18
